plot/plot.app.js

```diff
@@ -684,13 +690,42 @@
 
                                                     data.forEach(function (row) {
                                                         if(label) {
-                                                            values.labels.push(row[trace.legend_col]);
+                                                            var traceLabel=row[trace.legend_col];
```

Comment:
Fetching the markdown pattern for legends if available, render the markdown templating with a link tag and adding it to legend_markdown_pattern array so that the link can be used on click of legend event.